frontend/src/components/pages/CampOverview/index.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  AspectRatio,
+  Box,
+  Container,
+  Flex,
+  HStack,
+  Image,
+  Tag,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import placeHolderImage from "../../../assets/germany.jpeg";
+import costIcon from "../../../assets/coin.svg";
+import locationIcon from "../../../assets/location.svg";
+import ageIcon from "../../../assets/person.svg";
+import { Camp } from "../../../types/CampsTypes";
+import CampsAPIClient from "../../../APIClients/CampsAPIClient";
+
+export type CampOverviewProps = {
+  campId: string;
+};
+
+const CampOverview = (): JSX.Element => {
+  const [camp, setCamp] = useState<Camp>();
+  const campId = "62c098e7b4a7a433a7622ff4"; // hardcoded for now, TODO: update this
+
+  // TODO: update the statuses
+  enum Status {
```

Comment:
Can we call this `CampStatus` instead so not to confuse it with `UserStatus` in otherr components?